### ISSUE TITLE
PWA Banner

### ISSUE THREAD

**Issue by MinhxNguyen7** [icssc/AntAlmanac#936]
Add a banner/popup/button for people to install AA as a PWA so that they don't have to look for it in their kebab menu.

How-to for Chromium: https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/How_to/Trigger_install_prompt

**Comment by MinhxNguyen7** [comment]
How's this looking? Can you put up a draft PR?

**Comment by xgraceyan** [comment]
Yep, just opened a PR -- sorry for the delay, I was unfortunately busier than usual this week.
